azurerm/internal/services/kusto/kusto_eventhub_data_connection_resource.go

```diff
@@ -83,6 +83,16 @@ func resourceKustoEventHubDataConnection() *schema.Resource {
 				ValidateFunc: eventhubValidate.EventHubID,
 			},
 
+			"event_system_properties": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
```

Comment:
does this need to be Computed?

```suggestion
```

Reply:
Yep, since some of them are set implicitly and generated a diff otherwise.